lib/extend.es6

```diff
@@ -15,6 +15,7 @@ export default (options = {}) => {
         options.encoding = options.encoding || 'utf8';
         options.root = options.root || './';
         options.plugins = options.plugins || [];
+        options.strict = Object.prototype.hasOwnProperty.call(options, 'strict') ? !!options.strict : true;
```

Comment:
I think by default it is necessary to put `false`. this can cause errors in other users

Reply:
`strict` is currently true by default i.e. block names are checked. I'm happy to change it to not checking by default (i.e. `strict: false`), but that's a breaking change.

Reply:
Perhaps you are right)